base/uk.ac.stfc.isis.ibex.scriptgenerator/python_support/script_definition_loader.py

```diff
@@ -479,7 +479,11 @@ def getScriptDefinitions(self) -> list:
 
     def convert_list_of_actions_to_python(self, list_of_actions) -> List[Dict[AnyStr, AnyStr]]:
         python_list_of_actions: List = ListConverter().convert(list_of_actions, gateway._gateway_client)
-        return [MapConverter().convert(action, gateway._gateway_client) for action in python_list_of_actions]
+
+        # changes the order of the key:val pairs within the dictionaries it stores in the list
+        # return [MapConverter().convert(action, gateway._gateway_client) for action in python_list_of_actions]
```

Comment:
Did you mean to leave this commented out return in here?

Reply:
Yeah so from what I could tell from debugging it isn't needed. The outputs are the same just if you return the MapConverter one it messes up the order of the dict. I left it in just in case it broke something and needs to be added back quickly.